nats.go

```diff
@@ -2507,6 +2507,19 @@ func (nc *Conn) IsConnected() bool {
 	return nc.isConnected()
 }
 
+// Servers returns the list of known servers, including additional
```

Comment:
If they pass it in to us, yes I think its ok. We hold onto it for auto-reconnect.